jung-visualization/src/main/java/edu/uci/ics/jung/visualization/renderers/GradientVertexRenderer.java

```diff
@@ -66,17 +62,18 @@ public GradientVertexRenderer(
     this.pickedColorTwo = pickedColorTwo;
     this.pickedState = vv.getPickedVertexState();
     this.cyclic = cyclic;
-    this.layout = vv.getGraphLayout();
-    this.renderContext = vv.getRenderContext();
+    //    this.layout = vv.getGraphLayout();
+    //    this.renderContext = vv.getRenderContext();
```

Comment:
Ditto regarding earlier comment on commenting-out code.

For all places where you've done this, please consider whether they really need to be commented out, and if instead they can be uncommented again or simply removed.